source/loader/layers/sanitizer/asan_shadow.hpp

```diff
@@ -16,6 +16,8 @@
 #include "common.hpp"
 #include <unordered_set>
 
+#define NULLPTR_REDZONE_SIZE (4096)
```

Comment:
add a comment about why choose this size

Reply:
this macro needn't be in header file

Reply:
I choose this size becuase it is a common page value, and I don't have any other good reasons for it.

Reply:
You can query the size of page by GetVirtualMemGranularity.

Reply:
I don't think that is necessary. Even when we have the page size, how many pages should we use? I think some simple value is good enough for now.

Reply:
> I don't think that is necessary. Even when we have the page size, how many pages should we use?
I think it's reasonable to use 1 page, so that we don't waste the mapped physical memory.

Please use constexpr.